cmd/cleanup-jobs.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"context"
+	"sync"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/batch/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// CleanupJobs removes all failed jobs in a given namespace
+var CleanupJobs = &cobra.Command{
+	Use:   "cleanupjobs",
```

Comment:
Can we move this to `karina cleanup jobs` - It does conflict with `karina cleanup` which we should probably rename to `karina terminate-cluster`